changelog/unreleased/enhanchement-oc-tag-size

```diff
@@ -0,0 +1,5 @@
+Enhancement: Apply size property to oc-card
```

Comment:
post-merge nag: this was for oc-tag, not oc-card

Reply:
Ooooops
https://github.com/owncloud/owncloud-design-system/pull/2013